### DISCUSSION TITLE
[@types/tailwindcss] resolveConfig type issue

### DISCUSSION THREAD

**Discussion by kaguya3222** [DefinitelyTyped/DefinitelyTyped#61036] [Issues with a @types package]
Library: `@types/tailwindcss`
Version: `3.0.11`

This code gives me an error (but i don't know why I think it should work properly)

```
import resolveConfig from "tailwindcss/resolveConfig";
import tailwindConfig from "../tailwind.config.js";

const fullConfig = resolveConfig(tailwindConfig);
```

[Repository with reproduction](https://github.com/kaguya3222/types-tailwindcss-issue)

**UPDATE**: it works if I replace `/** @type {import('tailwindcss').Config} */` with `/** @type {import('tailwindcss/tailwind-config').TailwindConfig} */` in tailwind.config.js, but when i try to get `fullConfig.theme.fontSize?.body` i get error: 

```
Property 'body' does not exist on type 'TailwindThemeFontSizes'.
  Property 'body' does not exist on type '(getters: { theme: any; negative: any; colors: any; breakpoints: any; }) => TailwindValuesFontSize'.
```

**UPDATE V2.0**: I've found workaround: `const fontSize = fullConfig.theme?.fontSize as TailwindValuesFontSize;`

[a comment by an automated account is not shown]

**Comment by berghall** [comment]
tailwindcss seems to bundle their own types as of recently, so you might want to open an issue in [their official repo](https://github.com/tailwindlabs/tailwindcss) instead
